src/dbspro/cli/run.py

```diff
@@ -11,8 +11,8 @@
 
 logger = logging.getLogger(__name__)
 
-accepted_file = 'reads.fastq.gz'
-accepted_file_ext = '.fastq.gz'
+accepted_file = "reads.fastq.gz"
+accepted_file_ext = ".fastq.gz"
```

Comment:
These variable names should be capitalized as they are constants i.e. `ACCEPTED_FILE`  and `ACCEPTED_FILE_NAME`

Reply:
Sounds reasonable, we should probably fix that in BLR as well.

Reply:
I ended up
- moving the extension check to init 
- removing the accepted file

Since if `dbspro init` succeeds this should be correct.

Reply:
True!